packages/core/content-releases/admin/src/modules/releaseSlice.ts

```diff
@@ -3,15 +3,15 @@ import { createApi } from '@reduxjs/toolkit/query/react';
 import { pluginId } from '../pluginId';
 import { axiosBaseQuery } from '../utils/data';
 
-import type { CreateRelease, GetAllReleases } from '../../../shared/contracts/releases';
+import type { CreateRelease, GetReleases } from '../../../shared/contracts/releases';
 
 const releaseApi = createApi({
   reducerPath: pluginId,
   baseQuery: axiosBaseQuery,
   tagTypes: ['Releases'],
   endpoints: (build) => {
     return {
-      getRelease: build.query<GetAllReleases.Response, undefined>({
+      getRelease: build.query<GetReleases.Response, undefined>({
```

Comment:
@simotae14 shouldn't this be `getReleases` to fetch all releases? If so could I leave this to you to update with CS-298?